src/js/view/controls/next-display-icon.ts

```diff
@@ -1,21 +1,25 @@
 import UI from 'utils/ui';
+import type { CoreModel, PlayerAPI, PlaylistItemType } from 'types/generic.type';
 
 export default class NextDisplayIcon {
```

Comment:
same q about using this class as a type vs creating an interface and how that works because i truly don't know